src/gobexport/exporter/dat.py

```diff
@@ -84,6 +84,25 @@ def _to_number(value, precision=None):
         .replace('.', ',')
 
 
+def _to_number_zero(value, precision=None):
+    """Convert to number
+
+    Any zero result will be replaced by an empty string
+
+    Example:
+        0 => ''
+        0.0 => ''
+        0.0000 => ''
+        2.5 => 2,5
+
+    :param value:
+    :param precision:
+    :return:
+    """
+    result = _to_number(value, precision)
+    return '' if re.match(r'^0*(,0*)?$', result) else result
```

Comment:
Why not just try to cast to a float and check the result? Much more readable..

Reply:
OK, will change.